README.md

```diff
@@ -1,181 +1,134 @@
-# Chrome Apps samples
+# Chrome Extensions samples
 
-Official samples for Chrome Apps platform. If you want to learn about the platform, you can:
+Official samples for Chrome Extensions (and the [deprecated](https://developer.chrome.com/apps/migration) Chrome Apps platform).
 
-1. look at the source code of the samples below. Most samples have a "Try it now" button that allows you to install and play with it.
-2. read the [official docs](http://developer.chrome.com/apps)
-3. follow the [official Codelab](http://goo.gl/UHCS8) presented at Google I/O
+For more information, see [Chrome Developers](https://developer.chrome.com).
 
-If you have questions, [search](http://stackoverflow.com/questions/tagged/google-chrome-app) or [ask at StackOverflow](http://stackoverflow.com/questions/ask?tags=google-chrome-app) (observe the google-chrome-app tag) or join the [Chromium Apps](https://groups.google.com/a/chromium.org/forum/?fromgroups#!forum/chromium-apps) Google group.
+## Samples
 
+The directory structure is as follows:
 
-# Samples
+* api/ - trivial extensions focused on a single API package
```

Comment:
```suggestion
* api/ - extensions focused on a single API package
```

just thinking about this from an inclusive language perspective. I think words like 'trivial' or 'simple' may be unwelcoming to beginners (I know that's not the intent of the use of trivial here, but I think it can be read that way).

Reply:
This is imported from before. Happy to make the change.